example/ios/Podfile.lock

```diff
@@ -411,7 +411,7 @@ DEPENDENCIES:
   - React-RCTVibration (from `../node_modules/react-native/Libraries/Vibration`)
   - React-runtimeexecutor (from `../node_modules/react-native/ReactCommon/runtimeexecutor`)
   - ReactCommon/turbomodule/core (from `../node_modules/react-native/ReactCommon`)
-  - "RNFlashList (from `../../node_modules/@shopify/flash-list`)"
```

Comment:
question: was there a reason that we had `../../node_modules/@shopify/flash-list` previously? Is it to prevent the libary `react-native-collapsible-tab-view` from crashing in case that people do not have the dep installed or something?

Reply:
Not sure.

This is just the example app so it won't affect the library when published, so it should be fine.

Reply:
made it work I think, pushing changes in a bit

Reply:
will also add some videos